### PR TITLE
Updated ms.topic type

### PR DESCRIPTION
**Pull request by surbhigupta12**
Updated the ms.topic type for various files.

### CODE REVIEW

File: msteams-platform/bots/calls-and-meetings/debugging-local-testing-calling-meeting-bots.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: How to develop calling and online meeting bots on your local PC
```
Comment (heath-hamilton) [inline msteams-platform/bots/calls-and-meetings/debugging-local-testing-calling-meeting-bots.md:2]:
```suggestion
title: Debug your calling and meeting bot locally
```

File: msteams-platform/concepts/authentication/authentication.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Authenticate a user in Teams
```
Comment (heath-hamilton) [inline msteams-platform/concepts/authentication/authentication.md:2]: Authenticating app users

File: msteams-platform/concepts/deploy-and-publish/appsource/publish.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Microsoft Teams app approval submission process guidance
```
Comment (heath-hamilton) [inline msteams-platform/concepts/deploy-and-publish/appsource/publish.md:2]:
```suggestion
title: Understanding the Teams store submission process
```

File: msteams-platform/concepts/deploy-and-publish/appsource/publish.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
...
 description: Describes the submission approval process for getting your app published to the Microsoft Teams app store
+ms.topic: conceptual
```
Comment (heath-hamilton) [inline msteams-platform/concepts/deploy-and-publish/appsource/publish.md:4]:
```suggestion
ms.topic: overview
```

File: msteams-platform/concepts/device-capabilities/native-device-permissions.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Request device permissions for your Microsoft Teams tab
```
Comment (heath-hamilton) [inline msteams-platform/concepts/device-capabilities/native-device-permissions.md:2]:
```suggestion
title: Request device permissions for your tab
```

File: msteams-platform/concepts/device-capabilities/native-device-permissions.md [hunk 6]
```diff
@@ -1,6 +1,7 @@
...
 description: How to update your app manifest in order to request access to native features that usually require user consent
+ms.topic: conceptual
```
Comment (heath-hamilton) [inline msteams-platform/concepts/device-capabilities/native-device-permissions.md:4]:
```suggestion
ms.topic: how-to
```

File: msteams-platform/resources/bot-v3/bots-create.md [hunk 7]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Create a bot
 description: Describes how to create bots in Microsoft Teams
+ms.topic: reference
```
Comment (heath-hamilton) [inline msteams-platform/resources/bot-v3/bots-create.md:4]:
```suggestion
ms.topic: how-to
```

File: msteams-platform/resources/bot-v3/bots-overview.md [hunk 8]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Add bots to Microsoft Teams apps
 description: Describes how to get started developing bots in Microsoft Teams
+ms.topic: reference
```
Comment (heath-hamilton) [inline msteams-platform/resources/bot-v3/bots-overview.md:4]:
```suggestion
ms.topic: conceptual
```

File: msteams-platform/resources/dev-preview/developer-preview-intro.md [hunk 9]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Developer Preview
 description: Describes the features in the Public Developer Preview of Microsoft Teams
+ms.topic: reference
```
Comment (heath-hamilton) [inline msteams-platform/resources/dev-preview/developer-preview-intro.md:4]:
```suggestion
ms.topic: conceptual
```

File: msteams-platform/samples/virtual-assistant.md [hunk 10]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Virtual Assistant for Microsoft Teams
 description: How to create Virtual Assistant bot and skills for use in Microsoft Teams
+ms.topic: conceptual
```
Comment (heath-hamilton) [inline msteams-platform/samples/virtual-assistant.md:4]:
```suggestion
ms.topic: how-to
```

File: msteams-platform/tabs/how-to/authentication/auth-flow-tab.md [hunk 11]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Authentication flow for tabs
 description: Describes authentication flow in tabs
+ms.topic: how-to
```
Comment (heath-hamilton) [inline msteams-platform/tabs/how-to/authentication/auth-flow-tab.md:4]:
```suggestion
ms.topic: conceptual
```

File: msteams-platform/tutorials/code-samples.md [hunk 12]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Microsoft Teams code samples
```
Comment (heath-hamilton) [inline msteams-platform/tutorials/code-samples.md:2]:
```suggestion
title: App code samples
```

File: msteams-platform/tutorials/code-samples.md [hunk 13]
```diff
@@ -1,6 +1,7 @@
...
 description: Links and descriptions of sample applications for the Microsoft Teams developer platform
+s.topic: tutorial
```
Comment (heath-hamilton) [inline msteams-platform/tutorials/code-samples.md:4]:
```suggestion
ms.topic: reference
```
To me, this is more of a collection of links than a guided tutorial.

File: msteams-platform/whats-new.md [hunk 14]
```diff
@@ -1,6 +1,7 @@
 ---
 title: What's new
 description: Describes all the new developer features in Microsoft Teams
+ms.topic: overview
```
Comment (heath-hamilton) [inline msteams-platform/whats-new.md:4]:
```suggestion
ms.topic: reference
```

**Review by heath-hamilton — CHANGES_REQUESTED** [review]
@surbhigupta12 Please let me know if you want to discuss my proposed changes.

**Review by KirtiPereira — APPROVED** [review]
Approved.

**Comment by surbhigupta12** [comment]
@heath-hamilton, I've incorporated your suggestions. Please approve the PR if everything is fine.

**Review by heath-hamilton — APPROVED** [review]

**Pull request merged**